cinnamon-take-screenshot@duracell80/files/cinnamon-take-screenshot@duracell80/cinnamon-take-screenshot.sh

```diff
@@ -74,10 +74,15 @@ else
     # ELSE USE UNIVERSAL SECONDS FOR FILE NAME
     TS=$(date +%s)
 
-   gnome-screenshot --area --file="${DIR_TGT}/screenshot-auto_${TS}.png"
-    if [[ $(compgen -c | grep -iw 'pix' | head -n1 | wc -l) == "0" ]]; then
-        exit
+    FILENAME="${DIR_TGT}/screenshot-auto_${TS}.png"
+
+    gnome-screenshot --area --file="$FILENAME"
+
+    if [[ ! -f "$FILENAME" ]]; then
+        exit # MAYBE ALERT THE USER THAT THE SCREEN SHOT WAS NOT CREATED?
+    elif [[ $(compgen -c | grep -iw 'pix' | head -n1 | wc -l) != "0" ]]; then
```

Comment:
Why not something like `elif [[ $(command -v pix &>/dev/null; echo $?) -eq 0 ]]; then`? 

```suggestion
    elif [[ $(command -v pix &>/dev/null; echo $?) -eq 0 ]]; then
```

Reply:
Yeah, I was tempted to do that change but I was afraid of going beyond the scope and messing too much the inner logic/understanding of the author (since I would have to update another part that uses the same condition to check that) 🤔

Perhaps it's still worth making that change since I believe using `command` is more efficient than using many pipes the way it is being done currently.